srcpkgs/zsh-fast-syntax-highlighting/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'zsh-fast-syntax-highlighting'
+pkgname=zsh-fast-syntax-highlighting
+version=1.55
+revision=1
+wrksrc="fast-syntax-highlighting-${version}"
+depends="zsh"
+short_desc="Feature-rich syntax highlighting for ZSH"
+maintainer="akhil <akhil.j@outlook.in>"
+license="BSD-3-Clause"
+homepage="https://github.com/zdharma-continuum/fast-syntax-highlighting"
+changelog="https://raw.githubusercontent.com/zdharma-continuum/fast-syntax-highlighting/master/changelog.md"
+distfiles="https://github.com/zdharma-continuum/fast-syntax-highlighting/archive/refs/tags/v${version}.tar.gz"
+checksum=d06cea9c047ce46ad09ffd01a8489a849fc65b8b6310bd08f8bcec9d6f81a898
+
+do_install() {
+	vmkdir usr/share/zsh/plugins/${pkgname}
+	vcopy . usr/share/zsh/plugins/${pkgname}
```

Comment:
> https://github.com/void-linux/void-packages/runs/7713259385?check_suite_focus=true#step:8:9

I don't think many of these files should be installed.

Reply:
I am not sure which are necessary files to copy.